modules/standard/IO.chpl

```diff
@@ -5965,6 +5962,111 @@ proc fileWriter.writeBits(x: integral, numBits: int) : void throws {
   try this.write(new _internalIoBits(x:uint(64), numBits:int(8)));
 }
 
+/*
+  Writes a single Unicode codepoint to a ``fileWriter``
+
+  :arg codepoint: Unicode codepoint to write
+
+  :throws UnexpectedEofError: Thrown if the write operation exceeds the
+                              ``fileWriter``'s specified range.
+  :throws SystemError: Thrown if the codepoint could not be written to the ```fileWriter``.
+*/
+proc fileWriter.writeCodepoint(codepoint: int) throws {
+  try this.write(new _internalIoChar(codepoint:int(32)));
```

Comment:
(just to mark, if `_internalIoChar` does end up changing to store an `int(64)` this and a few other instances of `int(32)` will need to be changed, otherwise it's fine)